pkg/policy/l4.go

```diff
@@ -230,12 +230,16 @@ func (l4 L4Filter) matchesLabels(labels labels.LabelArray) bool {
 
 // L4PolicyMap is a list of L4 filters indexable by protocol/port
 // key format: "port/proto"
-type L4PolicyMap map[string]L4Filter
+type L4PolicyMap struct {
```

Comment:
This is no longer a map, the structure documentation is not correct. I would suggest a refactor of the struct's name

Reply:
oh, I didn't think we called it Map because it was a go map (L3Policy map is also not a map). I actually thought it described the usage as a mapping of proto/port to a rule. I would prefer to drop the Map in both L4PolicyMap and L3PolicyMap if they don't mean anything.